### PR TITLE
get files connector in slack

### PR DESCRIPTION
**Pull request by kakasoo**
# Test before submitting

Before submitting a Pull Request, please test your code. If you created a new created a new feature, then create the test function, too. Please refer to [this document](../CONTRIBUTING.md).

Put the name of the test function of the service you created in the command below.
This is a regular expression that returns all tests containing the character string, so if there are several test codes, you can put a keyword including all of the test codes and turn them.
If you omit the include statement and operate the test, all the test codes we write will work, and myriad errors will occur in your code without environmental variables.

```ts
npm run build && npm run test -- --include TEST_FUNCTION_NAME
```

Of course you won't be able to provide us with your environmental variables, so we'll replace them with our environmental variables to run the program and merge it depending on whether it passes or not.

# Purpose

Please explain to our developers what the features you modified or developed are for.

# Check List

- [ ] Passing an existing or newly created test
- [ ] sufficient summary and description for LLM to understand
- [ ] Does it ensure backward compatibility with existing features, or does the modifications not affect pre-modification request, response type?

### CODE REVIEW

File: src/api/structures/connector/slack/ISlack.ts [hunk 1]
```diff
@@ -770,4 +785,356 @@ export namespace ISlack {
...
+    created: number;
+
+    /**
+     * @title timestamp
+     */
+    timestamp: number;
+
+    /**
+     * @title name
+     * @example "tedair.gif"
```
Comment (seunghwako) [inline src/api/structures/connector/slack/ISlack.ts:807]: example이란게 원래 있나요? 단순 궁금증입니다.

File: src/api/structures/connector/slack/ISlack.ts [hunk 2]
```diff
@@ -770,4 +785,356 @@ export namespace ISlack {
...
+     */
+    ims: string[];
+
+    /**
+     * @title comments_count
+     */
+    comments_count: number & tags.Type<"uint64">;
+  }
+
+  export interface IGetFileOutput {
```
Comment (seunghwako) [inline src/api/structures/connector/slack/ISlack.ts:992]: description 필요할 것 같습니다.

File: src/api/structures/connector/slack/ISlack.ts [hunk 3]
```diff
@@ -206,8 +213,16 @@ export namespace ISlack {
...
+     * You can enter the ID value of the user who wants to look up the details.
      */
-    userIds: ISlack.User["id"][];
+    userIds: (ISlack.User["id"] &
+      Prerequisite<{
+        method: "post";
+        path: "/connector/slack/get-users";
+        jmesPath: "users[].{value: id, label: display_name}";
+      }>)[] &
+      tags.MinItems<1>;
```
Comment (seunghwako) [inline src/api/structures/connector/slack/ISlack.ts:225]: Array<ISlack.User["id"] &
      Prerequisite<{
        method: "post";
        path: "/connector/slack/get-users";
        jmesPath: "users[].{value: id, label: display_name}";
      }>>&
      tags.MinItems<1>;
      
      이렇게 쓰는거랑 차이점이 있나요?

File: src/providers/connector/slack/SlackProvider.ts [hunk 4]
```diff
@@ -209,18 +210,32 @@ export class SlackProvider {
...
-      const fields = this.getUserProfileFields(res.data.profile);
-      response.push({ ...res.data.profile, fields, id: userId });
+        if (typia.is<{ ok: false; error: "ratelimited" }>(res.data)) {
+          throw new Error(res.data.error);
+        }
+
+        const fields = this.getUserProfileFields(res.data.profile);
+        return { ...res.data.profile, fields };
+      } catch (err) {
+        throw err;
```
Comment (seunghwako) [inline src/providers/connector/slack/SlackProvider.ts:232]: JSON.stringfy

File: src/api/structures/connector/slack/ISlack.ts (hunk 3 above)
Comment (kakasoo) [reply src/api/structures/connector/slack/ISlack.ts:225]: Array로 감싸나 `[]` 를 사용하나 타입 상 같습니다!

Comment (seunghwako) [reply src/api/structures/connector/slack/ISlack.ts:225]: 넵 전체적으로 혼재된 것 같아서 나중에 시간나면 통일 한번 하시죠~

File: src/api/structures/connector/slack/ISlack.ts (hunk 1 above)
Comment (kakasoo) [reply src/api/structures/connector/slack/ISlack.ts:807]: 원래 JSDoc 스펙 상으로는 있습니다.
개발자들이 이게 뭘 의미하는 건지 헷갈릴까봐 일단 기입해뒀는데요, nestia 스펙에서는 반영 안됩니다.

**Pull request merged**